test/controllers/EditSetControllerSpec.scala

```diff
@@ -945,6 +946,18 @@ class EditSetControllerSpec extends BaseSpec {
               ExpectedSelectOption("1", "The National Archives, Kew", selected = true),
               ExpectedSelectOption("2", "British Museum, Department of Libraries and Archives"),
               ExpectedSelectOption("3", "British Library, National Sound Archive")
+            ),
+            relatedMaterial = Seq(
```

Comment:
But there should be 3 items; the one without the link is missing.

Reply:
Absolutely, I've changed the testing strategy now.